.pre-commit-config.yaml

```diff
@@ -1,28 +1,38 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v5.0.0
+    rev: "v5.0.0"
     hooks:
       - id: check-added-large-files
       - id: check-case-conflict
       - id: check-json
       - id: check-merge-conflict
-        args: ['--assume-in-merge']
+        args:
+          - '--assume-in-merge'
       - id: check-toml
       - id: check-yaml
       - id: end-of-file-fixer
       - id: mixed-line-ending
-        args: ['--fix=lf']
+        args:
+          - '--fix=lf'
       - id: sort-simple-yaml
       - id: trailing-whitespace
+
+  - repo: https://gitlab.com/vojko.pribudic.foss/pre-commit-update
+    rev: "v0.6.1"
+    hooks:
+      - id: pre-commit-update
+
```

Comment:
um. I think I would prefer to use https://pre-commit.ci/ as we do in other repos (need to figure why it was not enabled here).

It feels a cleaner/clearer workflow: a PR gets open and you can review the changes/failures -> commit gets added to main.

Reply:
Created https://github.com/mozilla-ai/any-agent/issues/119